### DISCUSSION TITLE
Assistant à côté du contenu et pas au dessus.

### DISCUSSION THREAD

**Discussion by aaalizk** [boscop-fr/assistant-rgaa#38] [Ideas]
Il serait plus pratique qu'au lieu de s'afficher par-dessus la page, l'encart de l'assistant décale le contenu sur le côté comme l'inspecteur de code pour ne plus passer au-dessus des éléments de la page.

![Capture](https://user-images.githubusercontent.com/69298848/156599156-bfa10942-ff98-49d4-9c3b-46ef1c1c77e1.PNG)

**Comment by felixgirault** [comment]
À l'origine, on s'était posé la question d'intégrer l'extension dans un panneau des outils de développement plutôt que dans la page.
Je pense toujours que ce serait une meilleure option, mais je suis probablement biaisé par mes habitudes de développeur ;)

**Comment by felixgirault** [comment]
Depuis la version 2, l'assistant utilise les sidebars natives.
